doc/examples.rst

```diff
@@ -76,3 +76,80 @@ Or just one of them:
 
    $ tt start demo:master
    • Starting an instance [master]...
+
+Working with application templates
+----------------------------------
+
+For example, we want to create an application template. In order to do this, create
+a directory, which will be used as a templates storage - ``./templates``. Then
+create an application template directory in it: ``./templates/simple`` with
+the content:
+
+``init.lua.tt.template``:
+
+.. code-block:: lua
+
+    local app_name = {{.name}}
+    local login = {{.user_name}}
+
+    require("fiber").sleep(1)
+
+``MANIFEST.yaml``:
+
+.. code-block:: yaml
+
+    description: Simple app
+        vars:
+            - prompt: User name
+            name: user_name
+            default: admin
+            re: ^\w+$
+
+Create a symlink in the template directory:
+``{{.name}} -> init.lua``
```

Comment:
What does it mean? this is example, describe clearly.
e.g. Create a symlink in the template directory:
`` sh
ln -s target link_name
``

Reply:
Removed